src/main/java/domain/numberscontainer/LottoNumbersDtoAssembler.java

```diff
@@ -0,0 +1,29 @@
+package domain.numberscontainer;
+
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/*
+ * LottoNumbersDto 생성 담당 클래스
+ */
+public class LottoNumbersDtoAssembler {
+    private static final String SPLIT_REGEX = ",";
+
+    public static LottoNumbersDto assemble(String numbers) {
+        return new LottoNumbersDto(parseSet(numbers));
+    }
+
+    public static LottoNumbersDto assemble(Set<LottoNumber> numbers) {
+        return new LottoNumbersDto(numbers);
+    }
+
+    private static LinkedHashSet<LottoNumber> parseSet(String numbersInput) {
```

Comment:
```suggestion
    private static Set<LottoNumber> parse(String rawNumbers) {
```

인터페이스 타입을 참조하면 어떠한 차이가 있을지 고민해보아요.
이펙티브 자바 `아이템 64. 객체는 인터페이스를 사용해 참조하라`를 참고해주세요.

또한 이름에 Set과 같은 자료형을 네이밍에 넣는 것은 권장하고 있지 않습니다!